sdk/inc/azure/iot/az_iot_common.h

```diff
@@ -61,6 +61,18 @@ typedef enum
   AZ_IOT_STATUS_TIMEOUT = 504,
 } az_iot_status;
 
+/**
+ * @brief Supported IoT Hub message properties
+ */
+#define AZ_IOT_HUB_MESSAGE_PROPERTIES_MESSAGE_ID \
+  "%24.mid" /**< Add unique identification to a message */
+#define AZ_IOT_HUB_MESSAGE_PROPERTIES_CORRELATION_ID \
+  "%24.cid" /**< Used in distributed tracing. More information here: \
+https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-distributed-tracing */
+#define AZ_IOT_HUB_MESSAGE_PROPERTIES_CONTENT_TYPE \
+  "%24.ct" /**< URL encoded and of the form text%2Fplain or application%2Fjson, etc */
+#define AZ_IOT_HUB_MESSAGE_PROPERTIES_CONTENT_ENCODING "%24.ce" /**< utf-8, utf-16, etc */
```

Comment:
nit: These should be capitalized.
```suggestion
#define AZ_IOT_HUB_MESSAGE_PROPERTIES_CONTENT_ENCODING "%24.ce" /**< UTF-8, UTF-16, etc */
```